qprofiler2/test/org/qcmg/qprofiler2/summarise/KmersSummaryTest.java

```diff
@@ -214,7 +212,7 @@ public void toXmlTest() throws IOException, DOMException, ParserConfigurationExc
 	
 	
 	@Test
-	public void bothForwardTest() throws IOException {
+	public void bothForwardTest() throws IllegalArgumentException, IOException {
```

Comment:
IllegalArgumentException is an unchecked exception and so you don't need it in the throws clause